src/Scaffold/Console/command/command.stub

```diff
@@ -9,7 +9,7 @@ class {{studly_name}} extends Command
     /**
      * @var string The console command name.
      */
-    protected $name = '{{lower_plugin}}:{{lower_name}}';
+    protected static $defaultName = '{{lower_plugin}}:{{lower_name}}';
```

Comment:
I think we should probably include the signature property like Laravel is doing in their scaffolding

Reply:
Sounds good.